sql/core/src/test/scala/org/apache/spark/sql/DataFrameWindowFunctionsSuite.scala

```diff
@@ -700,6 +700,42 @@ class DataFrameWindowFunctionsSuite extends QueryTest
         Row("b", 3, null, null, null)))
   }
 
+  test("lead with ignoreNulls") {
```

Comment:
can we add SQL tests as well?

Reply:
I will consider it and unify the support for [IGNORE NULLS | RESPECT NULLS] in the SQL parser.